docs/dev/developer-guides/transactions/transactions.md

```diff
@@ -92,4 +102,23 @@ Instead of signing the RLP-encoded transaction, the user signs the following typ
 | factoryDeps            | `bytes32[]` |
 | paymasterInput         | `bytes`     |
 
-These fields are handled by our [SDK](../../../api/js/features.md).
\ No newline at end of file
+These fields are handled by our [SDK](../../../api/js/features.md).
+
+### Priority: `0xff`
+
+This is a zkSync Era specific transaction type related to a L1 -> L2 transaction.
+
+??MORE HERE PLEASE??
+
+## Transaction statuses
+
+Transactions are always in one of the following statuses:
+
+- `Pending`: Not yet included in a block. 
+- `Included`: Included in a block but not yet verified.
+- `Verified`: Included in a block and verified. 
```

Comment:
The concept of verified is not clear. I ould expand on it so readers know what we are talking about

Reply:
OK, added something, give me 10 mins to push ..